assets/src/blocks/amp-story/edit.js

```diff
@@ -79,7 +84,12 @@ class EditPage extends Component {
 		} );
 
 		if ( IMAGE_BACKGROUND_TYPE === mediaType ) {
+			const featuredMedia = select( 'core/editor' ).getEditedPostAttribute( 'featured_media' );
```

Comment:
We're already using the `withSelect` HOC further down to pass some props. I'd move this there too.

Reply:
Good point, @swissspidy.

Also, I should have removed the logic in `amp-story/edit.js` entirely. The handler for the `MediaUpload` component conditionally updates the featured image:

<img width="897" alt="upload-media" src="https://user-images.githubusercontent.com/4063887/54726902-ad592e00-4b3a-11e9-8b32-c9cbaa12a520.png">


https://github.com/ampproject/amp-wp/pull/1998/commits/2237ad9c3add58b2b9c526bec55bb56016513938 removes the logic.